mqtt/mqttd/src/main.rs

```diff
@@ -19,30 +20,26 @@ async fn main() -> Result<(), Terminate> {
         .finish();
     let _ = tracing::subscriber::set_global_default(subscriber);
 
-    // TODO pass it to broker
-    // TODO make it an argument to override defaul config
-    let path: Option<String> = None;
-    let config = path
+    let matches = App::new("Edge MQTT Broker")
```

Comment:
nit: iotedged has this App and config init login in a separate module (https://github.com/Azure/iotedge/blob/master/edgelet/iotedged/src/app.rs). Not super important, just nit suggestion to keep main.rs smaller.

Reply:
moved out to a method